bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -55,275 +55,263 @@
+                </ul>
+                <ul class="videos">
+                  <li><a href="https://www.youtube.com/watch?v=SqEQCU_pvKs"><img src="{{ static('img/mozorg/about/manifesto/videos/SqEQCU_pvKs.jpg') }}" alt=""><span>{{ _('What is MozFest?') }}</span></a></li>
+                </ul>
+              </section>
+              <footer class="principle-footco">
+                <p class="share"><a class="tweet" href="{{ twitter_link('10', _('Magnifying the public benefit aspects of the Internet is an important goal, worthy of time, attention and commitment.')) }}">{{ _('Tweet this') }}</a></p>
+              </footer>
             </section>
           </div>
-          <footer>
-            <p><a href="{{ url('mozorg.about.manifesto-details') }}">{{ _('Read the entire manifesto') }}</a></p>
+          <footer class="content">
+            <p><a class="manifesto-details" href="{{ url('mozorg.about.manifesto-details') }}">{{ _('Read the entire manifesto') }}</a></p>
           </footer>
         </section>
-        <section id="sec-tell">
+        <!--
+        <section id="sec-tell" class="content">
           <h2>{{ _('Tell your friends') }}</h2>
           <div class="outer">
             <textarea maxlength="102">{{ _('I support Mozilla because… #mozmanifesto') }}</textarea>
             <p class="share"><a class="tweet" href="{{ twitter_link('custom', _('I support Mozilla because… #mozmanifesto'), '') }}">{{ _('Tweet this') }}</a></p>
           </div>
         </section>
-        <section id="sec-learn">
+        -->
+        <!--
+        <section id="sec-learn" class="content">
           <h2>{{ _('Learn more about…') }}</h2>
           <ul>
             <li><a href="{{ donate_url('mozillaorg_manifesto') }}">{{ _('Donating to Mozilla') }}</a></li>
             <li><a href="{{ url('mozorg.about.history') }}">{{ _('Our history') }}</a></li>
             <li><a href="{{ url('mozorg.mission') }}">{{ _('Our mission') }}</a></li>
           </ul>
         </section>
-        <section id="sec-newsletter">
-          {{ email_newsletter_form() }}
-        </section>
+        -->
+        <aside class="section section-newsletter" id="newsletter-subscribe">
+          <div class="content">
+            {% if LANG.startswith('en-') %}
+                {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Love the Web?'), subtitle=_('Get the Mozilla newsletter and help us keep it open and free.'), button_class='button-hollow button-light', spinner_color='#fff') }}
+              {% else %}
+                {{ email_newsletter_form(button_class='button-hollow button-light', spinner_color='#fff') }}
+              {% endif %}
+          </div>
+        </aside>
       </div>
-      <div id="mosaic-column-1" class="mosaic-column"></div>
-      <div id="mosaic-column-2" class="mosaic-column"></div>
     </div>
   </article>
 </main>
 
 {% endblock %}
 
-{% block email_form %}{% endblock %}
-
 {% block js %}
   <!--[if IE 9]>
```

Comment:
This conditional comment and script is included in the [pebbles base template](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-pebbles.html#L155-L157) by default, so we should be ok to remove this.